Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Settings.
+ARG USER_NAME=iriifservice
+ARG HOME_DIR=/$USER_NAME
+ARG SOURCE_DIR=/$HOME_DIR/source
+
+# Maven stage.
+FROM maven:3-openjdk-11-slim as maven
+ARG USER_NAME
+ARG HOME_DIR
+ARG SOURCE_DIR
+
+# Create the group (use a high ID to attempt to avoid conflits).
+RUN groupadd -g 3001 $USER_NAME
```

Comment:
As we are making significant improvements to the docker deployment, we should consider affording specifying the gid and uid as arguments to the image build. This will allow external permissions to be defined independently of this dockerfile. We do not want to have to adjust permissions based on static values in a Dockerfile. An example is for NFS file access, the permissions will be set and the deployed application should be configured to use the appropriate uid and gid as the permissions of the mount.